server/src/com/thoughtworks/go/server/service/PipelineConfigService.java

```diff
@@ -132,6 +136,17 @@ public void updatePipelineConfig(final Username currentUser, final PipelineConfi
         update(currentUser, pipelineConfig, result, updatePipelineConfigCommand);
     }
 
+    public List<Pipeline> pipelinesFor(Username username) {
```

Comment:
I don't think we need this method or the objects ```Pipeline```, ```Stage``` etc

There a method for getting pipeline groups of the user.
https://github.com/gocd/gocd/blob/e43774e9f991c2a0e2c82a3e41feeaa01a8efcbe/server/src/com/thoughtworks/go/server/service/SecurityService.java#L168

Can't we just get this from the controller and transform using the representer?